Package.swift

```diff
@@ -5,8 +5,6 @@ import PackageDescription
 
 let iceCppSettings: [CXXSetting] = [
     .define("ICE_BUILDING_SRC"),
-    .define("ICE_STATIC_LIBS"),
```

Comment:
Why do you remove ICE_STATIC_LIBS, isn't it required?

Reply:
It's not clear to me if this Package.swift build is a static build or not. Can you show the compile and link outputs?

Reply:
From my understaffing the Swift libraries we "export" can be either static or dynamic on the how they're being built.  So for example if the build is from Xcode on macOS it may choose do create Ice as a dynamic library and if for iOS it may do static. 

I'm not sure about the dependencies like our IceCpp target, I'll do more reading.

One issue with our ICE_STATIC_LIBS macro is that it excludes UDP and WS.

Reply:
I've added it back.